src/server.rs

```diff
@@ -374,6 +376,17 @@ impl Server {
             }
         }
     }
+
+    #[cfg(target_os = "macos")]
+    fn set_enable_retina(&self) {
+        let mut video_service_count = 0;
+        for (name, service) in self.services.iter() {
+            if Self::is_video_service_name(&name) && service.ok() {
+                video_service_count += 1;
+            }
+        }
+        *scrap::quartz::ENABLE_RETINA.lock().unwrap() = video_service_count < 2;
```

Comment:
So, the first one can be retina, then following subscription will be not-retina. If so, there is still issue for multiple window/display case, because the first display is retina, the other displays are not.

Reply:
Another worse case, the capture uses retina, but the subscriber's displays info shows non-retina.

Reply:
`scrap::quartz::ENABLE_RETINA` is a global control and is used when querying the display size in display service in a loop , so all retina or all non-retina

Reply:
count is the number of subscribed video services. When multiple connections are connected to the same screen, they will share the same video service, and count is 1.

Reply:
How about one subscribe to display 1, another one subscribe to display 2?

Reply:
we can ignore this case.

Reply:
video_service_count will be 2.

both display 1:
![image](https://github.com/rustdesk/rustdesk/assets/14891774/2da2acc8-8a92-477e-a627-db6d907b50a8)

one connection display 1 and another disply 2

![image](https://github.com/rustdesk/rustdesk/assets/14891774/d81d8bf3-c195-4458-aaed-ad1da6bf097a)

Reply:
> How about one subscribe to display 1, another one subscribe to display 2?

For this, retina is fine. But we can ignore this case.